### PR TITLE
Update coding guidelines for using View Bindings

### PR DESCRIPTION
**Pull request by maskaravivek**
Updating the doc based on the discussion in https://github.com/commons-app/apps-android-commons/issues/3530

### CODE REVIEW

File: android/Code-style.md [hunk 1]
```diff
@@ -66,23 +66,22 @@ Strings added must not have [unescaped HTML tags](https://github.com/commons-app
...
 
-```
-dependencies {
-    compile "com.jakewharton:butterknife:$BUTTERKNIFE_VERSION"
-    annotationProcessor "com.jakewharton:butterknife-compiler:$BUTTERKNIFE_VERSION"
-}
-```
+### Butterknife
+
+Contributors have the option of using [butterknife](https://github.com/JakeWharton/butterknife) for their view bindings. We recommend doing so if you can.
```
Comment (sivaraam) [inline android/Code-style.md]: Does this sentence kind of contradict the following sentence found above?

>  We have decided to move away from Butterknife and gradually replace its usage with [Kotlin Android Extensions ViewBinding](https://kotlinlang.org/docs/tutorials/android-plugin.html#view-binding).

Comment (macgills) [reply android/Code-style.md]: Do you have the availability to make a PR to delete the recommendation?

Comment (sivaraam) [reply android/Code-style.md]: Sure. Just so I understand correctly. You mean this: 

```suggestion
Contributors have the option of using [butterknife](https://github.com/JakeWharton/butterknife) for their view bindings.
```

Comment (sivaraam) [reply android/Code-style.md]: FYI: I've created #9 with the change.

**Pull request merged**